filebeat/module/mysql/slowlog/test/mysql-5.7.22.log-expected.json

```diff
@@ -3,85 +3,89 @@
         "@timestamp": "2018-08-07T08:27:47.000Z",
         "ecs.version": "1.0.0-beta2",
         "event.dataset": "mysql.slowlog",
+        "event.duration": 15000223000,
         "event.module": "mysql",
         "fileset.name": "slowlog",
         "input.type": "log",
         "log.flags": [
             "multiline"
         ],
         "log.offset": 41,
-        "mysql.slowlog.id": "7234",
-        "mysql.slowlog.ip": "218.76.8.37",
-        "mysql.slowlog.lock_time.sec": "0.000000",
+        "mysql.slowlog.id": 7234,
+        "mysql.slowlog.lock_time.sec": 0.0,
         "mysql.slowlog.query": "select sleep(15);",
-        "mysql.slowlog.query_time.sec": "15.000223",
-        "mysql.slowlog.rows_examined": "0",
-        "mysql.slowlog.rows_sent": "1",
-        "mysql.slowlog.timestamp": "1533630467",
-        "mysql.slowlog.user": "root"
+        "mysql.slowlog.query_time.sec": 15.000223,
+        "mysql.slowlog.rows_examined": 0,
+        "mysql.slowlog.rows_sent": 1,
+        "mysql.slowlog.timestamp": 1533630467,
+        "source.ip": "218.76.8.37",
+        "user.name": "root"
     },
     {
         "@timestamp": "2018-08-07T08:27:47.000Z",
         "ecs.version": "1.0.0-beta2",
         "event.dataset": "mysql.slowlog",
+        "event.duration": 153000,
         "event.module": "mysql",
         "fileset.name": "slowlog",
         "input.type": "log",
         "log.flags": [
             "multiline"
         ],
         "log.offset": 254,
-        "mysql.slowlog.host": "localhost",
-        "mysql.slowlog.lock_time.sec": "0.000061",
+        "mysql.slowlog.lock_time.sec": 6.1e-05,
         "mysql.slowlog.query": "SELECT count(*) FROM mysql.user WHERE user='root' and password='';",
-        "mysql.slowlog.query_time.sec": "0.000153",
-        "mysql.slowlog.rows_examined": "5",
-        "mysql.slowlog.rows_sent": "1",
-        "mysql.slowlog.timestamp": "1533630467",
-        "mysql.slowlog.user": "debian-sys-maint"
+        "mysql.slowlog.query_time.sec": 0.000153,
```

Comment:
Not for this PR but it's kind of add that we have `.sec` but then a micro second unit.

Reply:
Yeah. Well the unit is seconds, but there's more precision. So meh :-)